GettingStarted.md

```diff
@@ -2,29 +2,30 @@
 
 <!-- TOC -->
 
-- [What is Chocolatey?](#what-is-chocolatey)
-  - [Features](#features)
-  - [Common Thoughts](#common-thoughts)
-  - [Summary](#summary)
-  - [Already familiar with other package managers?](#already-familiar-with-other-package-managers)
-- [Requirements](#requirements)
-  - [Chocolatey Clients](#chocolatey-clients)
-    - [Chocolatey Components](#chocolatey-components)
-    - [Space Requirements](#space-requirements)
-    - [Memory Requirements](#memory-requirements)
-  - [Chocolatey Repository Servers](#chocolatey-repository-servers)
-  - [Chocolatey Central Management](#chocolatey-central-management)
-- [Using Chocolatey](#using-chocolatey)
-  - [Overriding default install directory or other advanced install concepts](#overriding-default-install-directory-or-other-advanced-install-concepts)
-- [Terminology](#terminology)
-- [What Are Chocolatey Packages?](#what-are-chocolatey-packages)
-- [How does Chocolatey work?](#how-does-chocolatey-work)
-  - [Installation](#installation)
-  - [Upgrade](#upgrade)
-  - [Uninstall](#uninstall)
-- [Where are Chocolatey packages installed to?](#where-are-chocolatey-packages-installed-to)
-- [How does Chocolatey work with Programs and Features? Existing installs?](#how-does-chocolatey-work-with-programs-and-features-existing-installs)
-- [Where does Chocolatey install packages from?](#where-does-chocolatey-install-packages-from)
+- [Getting Started](#getting-started)
```

Comment:
@steviecoaster was it the VSCode addin that automatically added this first level title to the ToC?